CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+# Amplication Contribution Guideline
+
+We're so glad that you're thinking about contributing to Amplication. We welcome your contributions!
+
+Here are some guidelines and instructions for contributing to our projects. Please go through them before making a contribution.
+
+## Code of Conduct
+
+Amplication governs its participants according to the Contributor [Code of Conduct](https://github.com/amplication/amplication/blob/master/code_of_conduct.md). As a contributor, you agree to uphold this code. 
+
+## FAQ and General Questions
+
+If your question concerns the code of a project, then you should create a [new issue] in the relevant project's repository.
+
+You can also join amplication chat channel on discord and then ask your question(s) in the relevant channel of our [Discord](https://discord.com/invite/Z2CG3rUFnu.
```

Comment:
Change the Discord link to https://amplication.com/discord

Reply:
Done